### ISSUE TITLE
Manager can invite contributors from indicator edit view #277

### ISSUE THREAD

**Issue by tmfrnz** [impactoss/impactoss-client#370]
As a category manager

I want to invite new users, make them contributors, and assign them to an indicator, when editing an indicator

requires https://github.com/impactoss/impactoss-server/issues/277

**Comment by ashbowe** [comment]
Could it also be done from the 'Users' page as well? That would give a bit of added flexibility and mean they could be set up if the admin knows they will be responsible for a few indicators and wants to batch assign them to some indicators.

**Comment by tmfrnz** [comment]
Yes this is the plan, see here: https://github.com/impactoss/impactoss-client/issues/371
